meinberlin/apps/votes/models.py

```diff
@@ -110,19 +110,8 @@ def is_valid_for_item(self, item):
 
     @staticmethod
     def get_hash_for_token(token, module):
-        params = argon2.profiles.RFC_9106_LOW_MEMORY
         salt = TokenSalt.get_or_create_salt_for_module(module)
-        # use per-modules salt as we can't store per-token salts
-        token_hash = argon2.low_level.hash_secret(
-            secret=token.encode(),
-            salt=salt.encode(),
-            time_cost=params.time_cost,
-            memory_cost=params.memory_cost,
-            parallelism=params.parallelism,
-            hash_len=params.hash_len,
-            type=params.type,
-            version=params.version,
-        ).decode()
+        token_hash = hashlib.sha3_512((salt + token).encode()).hexdigest()
```

Comment:
Is that what we already use for the passwords?

Reply:
for passwords we use PBKFD2 which uses sha2 I think, but hashlib is part of python which is nice , in contrast to the argon2 implementation which has not many stars on github